features/step_definitions/work_steps.rb

```diff
@@ -663,3 +680,9 @@
 Then /^the Remove Me As Chapter Co-Creator option should not be on the ([\d]+)(?:st|nd|rd|th) chapter$/ do |chapter_number|
   step %{I should not see "Remove Me As Chapter Co-Creator" within "ul#sortable_chapter_list > li:nth-of-type(#{chapter_number})"}
 end
+
+Then /^I should find myself on Chapter ([\d]+) of the work "([^\"]*)"$/ do |chapter_number, work_title|
+  step %{I should see "Chapter #{chapter_number}" within "div#chapters"}
```

Comment:
Inside `div#chapters` there's `div#chapter-1` if you want to be even more specific.

Or we could add a path:

```rb
when /^chapter (\d+) of the work "(.*?)"/
  work = Work.find_by(title: $2)
  chapter = work.chapters_in_order(include_content: false)[$1.to_i - 1]
  work_chapter_path(work, chapter)
```

then use it like `Then I should be on chapter 2 of the work "WIP"`.